src/Adapters/Storage/Mongo/MongoSchemaCollection.js

```diff
@@ -63,13 +63,18 @@ const defaultCLPS = Object.freeze({
 
 function mongoSchemaToParseSchema(mongoSchema) {
   let clps = defaultCLPS;
+  let indexes = {}
   if (mongoSchema._metadata && mongoSchema._metadata.class_permissions) {
     clps = {...emptyCLPS, ...mongoSchema._metadata.class_permissions};
   }
+  if (mongoSchema._metadata && mongoSchema._metadata.indexes) {
```

Comment:
`mongoSchema._metadata` is being checked twice for this and the above conditional. You could wrap both of these conditionals under 1 larger one, which just checks the metadata once.